### PR TITLE
Move the whole IPP in one package because it is a reusable flow now

### LINKED ISSUE #6243: Put all card reader presentation code in one package
Currently card reader flow is reusable across other flow, that's why having part of it in `prefs` package and the rest in `orders` doesn't make sense

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #6243 

### Description

IPP flow is now reusable so keeping part of it in `prefs` and the rest in `orders` is not right. This PR moves the whole presentation layer to 1 `cardreader` package

### Testing instructions

Just run the main IPP flows

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/cardreader/onboarding/CardReaderOnboardingFragment.kt [hunk 1]
```diff
@@ -133,14 +133,20 @@ class CardReaderOnboardingFragment : BaseFragment(R.layout.fragment_card_reader_
...
-    private fun showLoadingState(
-        view: View,
-        state: CardReaderOnboardingViewModel.OnboardingViewState.LoadingState
-    ) {
+    private fun showLoadingState(view: View) {
         val binding = FragmentCardReaderOnboardingLoadingBinding.bind(view)
-        UiHelpers.setTextOrHide(binding.textHeaderTv, state.headerLabel)
-        UiHelpers.setTextOrHide(binding.hintTv, state.hintLabel)
-        UiHelpers.setImageOrHideInLandscape(binding.illustrationIv, state.illustration)
+        UiHelpers.setTextOrHide(
```
Comment (malinajirka) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/cardreader/onboarding/CardReaderOnboardingFragment.kt]: 🔍 Is this change intentional?

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/cardreader/onboarding/CardReaderOnboardingViewModel.kt [hunk 2]
```diff
@@ -86,7 +86,10 @@ class CardReaderOnboardingViewModel @Inject constructor(
                     }
                 CardReaderOnboardingState.WcpayNotInstalled ->
```
Comment (malinajirka) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/cardreader/onboarding/CardReaderOnboardingViewModel.kt]: ❓ Changes in this file look differently than what I'd expect from the auto formatter - do you have line length set to the correct value?

**Review by malinajirka — APPROVED** [review]
Thanks @kidinov! Good idea to move everything together since it's now a single independent flow ❤️.

This PR is good to be merged, I've left two very minor comments. 

I'm just a bit worried it might create some conflicts with the ongoing work of other folks, wdyt? I've always avoided doing package changes on a code which is being worked on - perhaps, git is smart and it won't create any conflicts and just move the files. I'm not sure - do you have any experience with it? (cc  @AnirudhBhat @rossanafmenezes)

**Comment by AnirudhBhat** [comment]
> Thanks @kidinov! Good idea to move everything together since it's now a single independent flow ❤️.
> 
> This PR is good to be merged, I've left two very minor comments.
> 
> I'm just a bit worried it might create some conflicts with the ongoing work of other folks, wdyt? I've always avoided doing package changes on a code which is being worked on - perhaps, git is smart and it won't create any conflicts and just move the files. I'm not sure - do you have any experience with it? (cc @AnirudhBhat @rossanafmenezes)

Ooh. I touch a lot of these files for Interac refund and I'm afraid these changes might affect Interac refund PRs. I'm not quite sure if Android Studio makes it easier to fix issues with packages once I pull these changes. What do you all think?

**Comment by malinajirka** [comment]
@AnirudhBhat Perhaps, you could just attempt to merge this PR into your branch - I mean just as a 2 minutes POC to check how many conflict it creates if any - and revert it. Wdyt?

**Comment by kidinov** [comment]
@AnirudhBhat @malinajirka This is definitely less important than refund logic, so if this causes any delay, let's just wait and I'll handle all the conflicts after the refund PRs merge

**Comment by rossanafmenezes** [comment]
> Thanks @kidinov! Good idea to move everything together since it's now a single independent flow ❤️.
> 
> This PR is good to be merged, I've left two very minor comments.
> 
> I'm just a bit worried it might create some conflicts with the ongoing work of other folks, wdyt? I've always avoided doing package changes on a code which is being worked on - perhaps, git is smart and it won't create any conflicts and just move the files. I'm not sure - do you have any experience with it? (cc @AnirudhBhat @rossanafmenezes)

In my case, I think it's fine. I'm currently working on a test branch but only locally. Plus the manuals screen is less important.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/cardreader/onboarding/CardReaderOnboardingFragment.kt (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/cardreader/onboarding/CardReaderOnboardingFragment.kt]: Oh, it's actually AS being too smart. It noticed that `CardReaderOnboardingViewModel.OnboardingViewState.LoadingState` an object so there is no need to pass an instance but you can use it statically 

Reverted - 16af5d34e05861b764a673f05f99d07b658d82a0

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/cardreader/onboarding/CardReaderOnboardingViewModel.kt (hunk 2 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/cardreader/onboarding/CardReaderOnboardingViewModel.kt]: Reverted - b4f7f5d4fb62fdcf59bc0103e9b949ee4aceb2f5

**Review by malinajirka — APPROVED** [review]
Great job @kidinov, thanks for doing these types of cleanups/improvements! LGTM :shipit:

**Pull request merged**